lib/solidus_paypal_braintree/engine.rb

```diff
@@ -34,19 +40,13 @@ def self.activate
       paths["app/views"] << "lib/views/frontend"
     end
 
-    if SolidusSupport.backend_available?
+    if backend_available?
       config.assets.precompile += ["spree/backend/solidus_paypal_braintree.js"]
       paths["app/controllers"] << "lib/controllers/backend"
 
       # We support Solidus v1.2, which requires some different markup in the
       # source form partial. This will take precedence over lib/views/backend.
-      paths["app/views"] << "lib/views/backend_v1.2" if SolidusSupport.solidus_gem_version < Gem::Version.new('1.3')
-
-      # Solidus v2.4 introduced preference field partials but does not ship a hash field type.
-      # This is solved in Solidus v2.5.
-      if SolidusSupport.solidus_gem_version <= Gem::Version.new('2.5.0')
-        paths["app/views"] << "lib/views/backend_v2.4"
-      end
+      paths["app/views"] << "lib/views/backend_v1.2"
```

Comment:
Think we do not need these two backend views. As we only support Spree 2.4 in this gem we should go with the `lib/views/backend_v1.2` but rename them to `lib/views/backend`